static-analysis/lint/src/main/java/com/uber/autodispose/lint/AutoDisposeIssueRegistry.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2018. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.uber.autodispose.lint
+
+import com.android.tools.lint.client.api.IssueRegistry
+import com.android.tools.lint.detector.api.Issue
+
+class AutoDisposeIssueRegistry(override val issues: List<Issue> = listOf()) : IssueRegistry() {
```

Comment:
Let's use the autoservice registration for this

https://github.com/hzsweers/rxjava2-optionalcheckreturnvalue-checkers/blob/master/lint/src/main/kotlin/io/sweers/rxjava2optionalcheckreturnvaluechecker/lint/OptionalCheckReturnValueIssueRegistry.kt

Reply:
Nice! That lint registry always felt like the most awkward part.